### PR TITLE
[FIX-43] responsive chapters and main layout

### PR DESCRIPTION
**Pull request by ell-ska**

### CODE REVIEW

File: app/consent-form/_components/chapter/ChapterIcons.tsx [hunk 1]
```diff
@@ -621,17 +628,30 @@ const ConsentDisabled = () => {
 type ChapterIconsProps = {
   icon: Icon
   state: State
+  className?: string
 }
 
-const ChapterIcon = ({ icon, state }: ChapterIconsProps) => {
+const ChapterIcon = ({ icon, state, className }: ChapterIconsProps) => {
   if (icon === 'introduction') {
```
Comment (idamakela) [inline app/consent-form/_components/chapter/ChapterIcons.tsx:635]: Could a switch statement instead of else if's be implemented instead?

**Review by idamakela — CHANGES_REQUESTED** [review]

File: app/consent-form/_components/chapter/ChapterIcons.tsx (hunk 1 above)
Comment (ell-ska) [reply app/consent-form/_components/chapter/ChapterIcons.tsx:635]: I tried that first but it didn’t really work since it “switches” over multiple things

Comment (idamakela) [reply app/consent-form/_components/chapter/ChapterIcons.tsx:635]: Okay! Good to know!

**Review by idamakela — APPROVED** [review]

**Comment by ell-ska** [comment]
ready for testing

**Review by petksks — APPROVED** [review]

**Comment by idaakermark** [comment]
Testing on dev: responsivness looks good! Ready for production.

**Pull request merged**